### ISSUE TITLE
Display MViews in OEP

### ISSUE THREAD

**Issue by Ludee** [openego/data_processing#122]
At some points (e.g. borders schema) we create MViews of existing tables. In eGoDP this is used to select a **version** or change the SRID of the geom. These "tables" are NOT displayed online atm. And they don't have (maintained) metadata.

Here are my questions for discussion:
1. Do we want MViews to be displayed online?
2. Which rules apply when using MViews?
3. Do they need to be under "revision"?

**Comment by lukasol** [comment]
I do not think we need them to be displayed online since most (if not all?) of the data is at some point published in tables. That is for example true for the powerplant tables, which are basically just "translated" from an MView to a table.

**Comment by wolfbunke** [comment]
Good point but I think if a Mview is that impotent to be on the platform it  should be a database table or? But maybe @MGlauer has a good idea?

**Comment by gplssm** [comment]
There is currently no use case for me, thus I have no opinion.
